packages/admin-frontend/components/EditBook/EditTabBar.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2018 GDL
+ *
+ * See LICENSE
+ */
+import * as React from 'react';
+import Router from 'next/router';
+import { withRouter } from 'next/router';
+import AppBar from '@material-ui/core/AppBar/AppBar';
+import Tab from '@material-ui/core/Tab/Tab';
+import Tabs from '@material-ui/core/Tabs/Tabs';
+
+type Props = {
+  router: {
+    pathname: string,
+    query: {
+      id: string,
+      lang: string
+    }
+  }
+};
+
+type State = {};
+
+class RouteAwareTabBar extends React.Component<Props, State> {
+  handleChange = (event: Event, selectedTab: number) => {
```

Comment:
Rather than routing in a click handler, I think we should display the tabs as links instead.

Reply:
Still ended up with a click handler to get hold of the `selectedTab` value, but managed to set `pathname` for `Route` to be dynamic.